.travis.yml

```diff
@@ -12,6 +12,12 @@ otp_release:
 
 script:
   - make testing
+
+cache:
```

Comment:
Holy moly, I did not know this was a thing.  I am going to have to bring this over to other projects.

Reply:
@doomspork same 😄